lib/cambiatus_web/controllers/rich_link_controller.ex

```diff
@@ -10,6 +10,8 @@ defmodule CambiatusWeb.RichLinkController do
   alias CambiatusWeb.Resolvers.{Accounts, Commune, Shop}
   alias Cambiatus.Repo
 
+  @product_image "https://cambiatus-uploads.s3.amazonaws.com/cambiatus-uploads/b214c106482a46ad89f3272761d3f5b5"
```

Comment:
maybe we could be more explicit: 
```suggestion
  @default_product_image "https://cambiatus-uploads.s3.amazonaws.com/cambiatus-uploads/b214c106482a46ad89f3272761d3f5b5"
```